### ISSUE TITLE
Doesn't work with non-standard ssh port

### ISSUE THREAD

**Issue by adamsalter** [vigetlabs/capistrano_rsync_with_remote_cache#2]
I have a non standard ssh port set in my capistrano deploy task:

```
ssh_options[:port] = 1234
```

Unfortunately this capistrano task doesn't play nicely... all my tasks still run, but I can't deploy since rsync is not connecting on the correct port...

**Comment by adamsalter** [comment]
I was able to fix it by changing line 61:

```
    def rsync_options
      configuration[:rsync_options] || "-az --delete -e 'ssh -p #{ssh_options[:port] || 22}'"
    end
```

**Comment by mcornick** [comment]
respect ssh_options[:port] in rsync; closed by 2bfb0f6f6c8f696fa66f34a1dbe1e324bafa737b

**Comment by mcornick** [comment]
Thanks for the report. This should be fixed in version 2.3.6. Please let me know if you still have problems.
